# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/child/FileSystemManagerChild.h
# Commit: 302fdb94bf7f
# Full Hash: 302fdb94bf7f5cd817b69fc4eaaaf6ff8b62bec8
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2023-02-06 21:36:23
# Regressor Bug: 1809064
# File Overlap Count: 1
# Description:
#   Bug 1809064 - Use managed endpoints for PFileSystemAccessHandle construction; r=dom-storage-reviewers,jari
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D166342
# ==============================================================================

diff -r 100b45bcf622 -r 302fdb94bf7f dom/fs/child/FileSystemManagerChild.h
--- a/dom/fs/child/FileSystemManagerChild.h	Mon Feb 06 15:01:20 2023 +0000
+++ b/dom/fs/child/FileSystemManagerChild.h	Mon Feb 06 15:10:34 2023 +0000
@@ -30,9 +30,6 @@
 
   virtual void Shutdown();
 
-  already_AddRefed<PFileSystemAccessHandleChild>
-  AllocPFileSystemAccessHandleChild();
-
   already_AddRefed<PFileSystemWritableFileStreamChild>
   AllocPFileSystemWritableFileStreamChild();
 